readthedocs/core/management/commands/import_github.py

```diff
@@ -9,4 +9,6 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         if len(args):
             for slug in args:
-                import_github(user=User.objects.get(username=slug), sync=True)
+                service = GitHubService.for_user(User.objects.get(username=slug))
+                if service is not None:
+                    service.sync(sync=True)
```

Comment:
I didn't see any, this is brought over from before the refactor. Not sure what sync=false would ever be used for.